pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -181,7 +181,7 @@ stdenv.mkDerivation {
       ln -s $out/libexec/git-core/git-http-backend $out/bin/git-http-backend
     '' + stdenv.lib.optionalString perlSupport ''
       # wrap perl commands
-      makeWrapper "$out/share/git/contrib/credential/netrc/git-credential-netrc" $out/bin/git-credential-netrc \
+      makeWrapper "$out/share/git/contrib/credential/netrc/git-credential-netrc.perl" $out/bin/git-credential-netrc \
```

Comment:
Looks like the relevant upstream commit is this one: https://github.com/git/git/commit/1c78c78d2

This script now requires a `make` step -- `git-credential-netrc.perl` is just a source file (which is why it was renamed), and the ultimate executable is still named `git-credential-netrc`.

The existing `make` steps in `postBuild` may provide a useful model.